### ISSUE TITLE
Scheduler error

### ISSUE THREAD

**Issue by muthugit** [dagster-io/dagster#2277]
I am getting 
```
AttributeError: 'NoneType' object has no attribute 'all_schedules'
```
when run the scheduler

**Comment by newturok** [comment]
Same issues

**Comment by alangenfeld** [comment]
do you have any further stack trace? What version are you using?

**Comment by helloworld** [comment]
these errors should have been fixed in release 0.7.4 - if you're still hitting this after updating the stack trace would be helpful to debug the error.

**Comment by natekupp** [comment]
Hi all, closing given that this issue has been resolved—please re-open if you encounter again
